mods/inventory_icon/depends.txt

```diff
@@ -1 +0,0 @@
-unified_inventory
```

Comment:
Ok to delete depends.txt, but i don't see a mod.conf (mod.conf replace depends.txt if i guess right)

Reply:
`mod.conf` wasn't deleted, it's still there; check it out at the `update_3d_armor` branch.